### PR TITLE
feat(deployment): added digital ocean app platform

### PR DESCRIPTION
**Pull request by GabrielBottari**
#3

### CODE REVIEW

File: .do/remap-api-url.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const replace = require("replace-in-file")
+
+try {
+  console.log(
```
Comment (yann510) [inline .do/remap-api-url.js:4]: Do we want to log the output of the replacement?

File: README.md [hunk 2]
```diff
@@ -112,6 +112,25 @@ To have the CI working, you must:
 - Link your repository with [Coveralls](https://coveralls.io/repos/new).
 - (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
 
+### Deployment
```
Comment (yann510) [inline README.md:115]: This section should be at the very end of the file as it is the last thing you want to do.

File: apps/api/src/config/configuration.ts [hunk 3]
```diff
@@ -12,6 +12,7 @@ export const configuration = () => ({
     username: process.env.DATABASE_USERNAME || "postgres",
     password: process.env.DATABASE_PASSWORD || "postgres",
     synchronize: process.env.DATABASE_SYNCHRONIZE || true,
+    ca: process.env.DATABASE_CA_CERT,
```
Comment (yann510) [inline apps/api/src/config/configuration.ts:15]: Avoid acronyms as much as possible. Let's use the full name.

File: apps/api/src/main.ts [hunk 4]
```diff
@@ -24,7 +25,7 @@ async function bootstrap() {
   SwaggerModule.setup("documentation", app, document)
 
   const configService = app.get(ConfigService)
-  await app.listen(configService.get<number>("port"))
+  await app.listen(configService.get<number>("port"), "0.0.0.0")
```
Comment (yann510) [inline apps/api/src/main.ts:28]: This should be in the `configuration.ts` file

**Review by yann510 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (GabrielBottari) [reply README.md:115]: Its at the end of Usage 🦈

File: .do/remap-api-url.js (hunk 1 above)
Comment (GabrielBottari) [reply .do/remap-api-url.js:4]: image

Comment (GabrielBottari) [reply .do/remap-api-url.js:4]: Gives a nice log output to see if its was succesful

**Review by yann510 — APPROVED** [review]
GG

**Pull request merged**